### PR TITLE
Sign command returns signature as Buffer

### PR DESCRIPTION
**Pull request by ftheirs**
Sign command returns only one signature that should be included in all signable descriptions (Spends and Mints)

### CODE REVIEW

File: src/types.ts [hunk 1]
```diff
@@ -24,7 +24,7 @@ export interface ResponseProofGenKey extends ResponseBase {
 }
 
 export interface ResponseSign extends ResponseBase {
-  signatures?: Buffer[]
+  signatures?: Buffer
```
Comment (emmanuelm41) [inline src/types.ts]: Should it be called "signature" instead?

File: src/index.ts [hunk 2]
```diff
@@ -92,15 +90,11 @@ export default class IronfishApp extends GenericApp {
...
-          for (let i = 0; i < signaturesLen; i++) {
-            const tmpSignature = response.subarray(i * REDJUBJUB_SIGNATURE_LEN, i * REDJUBJUB_SIGNATURE_LEN + REDJUBJUB_SIGNATURE_LEN)
-            signatures.push(tmpSignature)
-          }
+        if (returnCode === LedgerError.NoErrors && response.length == (2 + REDJUBJUB_SIGNATURE_LEN)) {
+          const signature = response.subarray(0, REDJUBJUB_SIGNATURE_LEN)
 
           return {
-            signatures,
+            signature,
```
Comment (emmanuelm41) [inline src/index.ts:97]: I see it is correct here.

**Review by emmanuelm41 — CHANGES_REQUESTED** [review]
Just one comment important

File: src/types.ts (hunk 1 above)
Comment (ftheirs) [reply src/types.ts]: Good catch 😉 Fixed!

**Review by emmanuelm41 — APPROVED** [review]

**Pull request merged**